go.mod

```diff
@@ -22,6 +22,7 @@ require (
 	github.com/kyleconroy/sqlc v1.16.0
 	github.com/kylelemons/godebug v1.1.0
 	github.com/lib/pq v1.10.7
+	github.com/markbates/pkger v0.15.1
```

Comment:
Hmm, not sure why this is being added, perhaps an accident? I removed pkger in #1057

Reply:
thanks for pointing out! It stayed in my gomod because pkged.go file didn't get deleted since it's ignored in git, so it kept pulling the dep. Just manually cleaned up and it went away.